CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
To the best of my knowledge, you need `3.1.3` for `CMAKE_CXX_STANDARD`.  Or at least that's the first version it's actually documented in.

Reply:
Good catch! 👍